packages/atomic/cypress/e2e/search-box/search-box.cypress.ts

```diff
@@ -143,10 +143,11 @@ describe('Search Box Test Suites', () => {
             })
           )
           .init();
+        cy.wait(2000);
```

Comment:
Do we need the wait here? The test passes without it

Reply:
There were flaky. They would "eventually" pass by replaying the test after initial init, or if you are "lucky".

By adding a wait here it should be better,